packages/material-ui/src/ExpansionPanel/index.d.ts

```diff
@@ -1,2 +1,18 @@
-export { default } from './ExpansionPanel';
-export * from './ExpansionPanel';
+/**
+ * Use `Accordion` instead. This component will be removed in v6.
+ *
+ * @deprecated
+ */
+export { default } from '../Accordion';
+/**
+ * Use `AccordionProps` instead. This interface will be removed in v6.
+ *
+ * @deprecated
+ */
+export { AccordionProps as ExpansionPanelProps } from '../Accordion';
+/**
+ * Use `AccordionClassKey` instead. This type will be removed in v6.
```

Comment:
@oliviertassinari if we plan to merge this to master later on, should we update the version in these comments?

Reply:
Well spotted 👍